tests/test_ui_app.py

```diff
@@ -1683,6 +1683,159 @@ def test_auth_logout(self):
             self.assertTrue(
                 '<h2>Fedora Public Active Mirrors</h2>' in data)
 
+    def test_toggle_private_flag(self):
```

Comment:
I recommend to split these tests, so it is more clear, what are you trying to test.

Reply:
Thanks for the review. I split the one test into two different tests.